iolite/__init__.py

```diff
@@ -1 +1,5 @@
-__version__ = "0.1.0"
+from pathlib import Path
+
+from single_source import get_version
+
+__version__ = get_version(__name__, Path(__file__).parent.parent)
```

Comment:
Newline missing at the end of the file?

Reply:
Hmm - it should be there we have pre-commit ensuring of this